scripts/crypto.getPrice.js

```diff
@@ -10,5 +10,5 @@ try {
   const response = await axios.get(url, { params });
   return response.data[currency].usd;
 } catch (error) {
-  throw new Error(`Could not fetch price for ${currency}: ${error.message}`);
+  throw new Error("Could not fetch price for " + currency + ": " + error.message);
```

Comment:
A better way to solve this is to escape these characters. We do this within the shims codegen, see there.

Reply:
`code.replaceAll("`", "\\`").replaceAll("$", "\\$")`
https://github.com/polywrap/evo.ninja/blob/f0db569e624a2cd3704923145694caf18fd31e0d/packages/agent-utils/rollup.config.mjs#L19